cmd/internal/kube/run.go

```diff
@@ -0,0 +1,31 @@
+package kube
+
+import (
+	"os"
+
+	"github.com/postmanlabs/postman-insights-agent/cmd/internal/cmderr"
+	"github.com/postmanlabs/postman-insights-agent/cmd/internal/kube/daemonset"
+	"github.com/spf13/cobra"
+)
+
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run the Postman Insights Agent in your Kubernetes cluster as a daemon-set",
+	Long:  "Run the Postman Insights Agent in your Kubernetes cluster as a daemon-set to collect and send data to Postman Insights",
```

Comment:
Unfortunately Kubernetes convention is to avoid the hyphen: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

It is also to camel-case it as DaemonSet?